### PR TITLE
Blender: Use client query functions

### PR DESCRIPTION
**Pull request by iLLiCiTiT**
## Brief description
Blender is using query functions from [PR](https://github.com/pypeclub/OpenPype/pull/3288) instead of direct use of mongo queries.

## Description
Blender uses query functions from `openpype.client`.

## Testing notes:
1. Run blender
2. Frame start/end should be set to asset's resolution
3. Publish layout

PR waits for merge of [PR](https://github.com/pypeclub/OpenPype/pull/3288)

### CODE REVIEW

File: openpype/hosts/blender/plugins/publish/extract_layout.py [hunk 1]
```diff
@@ -131,43 +129,32 @@ def process(self, instance):
 
         fbx_count = 0
 
+        project_name = instance.context["projectEntity"]["name"]
```
Comment (simonebarbieri) [inline openpype/hosts/blender/plugins/publish/extract_layout.py:132]: I am getting this error:

```
Traceback (most recent call last):
  File "C:\Dev\OpenPype\openpype\.venv\lib\site-packages\pyblish\plugin.py", line 791, in __getitem__
    return next(c for c in self if c.id == item)
StopIteration

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Dev\OpenPype\openpype\.venv\lib\site-packages\pyblish\plugin.py", line 522, in __explicit_process
    runner(*args)
  File "C:\Dev\OpenPype\openpype\openpype\hosts\blender\plugins\publish\extract_layout.py", line 132, in process
  File "C:\Dev\OpenPype\openpype\.venv\lib\site-packages\pyblish\plugin.py", line 793, in __getitem__
    raise KeyError("%s not in list" % item)
KeyError: 'projectEntity not in list'
```

I think this should be `instance.data["projectEntity"]["name"]`?

Comment (iLLiCiTiT) [reply openpype/hosts/blender/plugins/publish/extract_layout.py]: Should be fixed

**Review by 64qam — DISMISSED** [review]
image

**Comment by iLLiCiTiT** [comment]
> image

This doesn't seem like an issue of this PR. Crash is not related to changes made here.

**Comment by simonebarbieri** [comment]
Works fine to me now
image

**Review by simonebarbieri — APPROVED** [review]

**Pull request merged**